scripts/tests/chiptest/test_definition.py

```diff
@@ -208,15 +211,37 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str):
 
         try:
             if self.target == TestTarget.ALL_CLUSTERS:
-                app_cmd = paths.all_clusters_app
+                target_app = paths.all_clusters_app
             elif self.target == TestTarget.TV:
-                app_cmd = paths.tv_app
+                target_app = paths.tv_app
             elif self.target == TestTarget.LOCK:
-                app_cmd = paths.lock_app
+                target_app = paths.lock_app
             else:
                 raise Exception("Unknown test target - "
                                 "don't know which application to run")
 
+            # The paths argument is a dataclass where each field contains a list of one application path
+            for field in paths.__dataclass_fields__:
+                path = getattr(paths, field)
+
+                # For the app indicated by self.target, give it the 'default' key to add to the register
+                if path == target_app:
+                    key = 'default'
+                else:
+                    key = os.path.basename(path[0])
+
+                app = App(runner, path)
+                # Add the App to the register immediately, so if it fails during
+                # start() we will be able to clean things up properly.
+                apps_register.add(key, app)
+                # Remove server application storage (factory reset),
+                # so it will be commissionable again.
+                app.factoryReset()
+                # Only start the app indicated by self.target
+                if key == 'default':
+                    default_app = app
+                    app.start({'--KVS': '/tmp/chip_kvs_default'})
```

Comment:
Remove
```
# Only start the app indicated by self.target
if key == 'default':
         default_app = app
         app.start({'--KVS': '/tmp/chip_kvs_default'})
```

And next to `# Only pair the app indicated by self.target` use: 
```
app = apps_register.get('default')
app.start({'--KVS': '/tmp/chip_kvs_default'})
```

Reply:
Done